quarkus.jdt/com.redhat.quarkus.jdt.core/src/main/java/com/redhat/quarkus/jdt/core/JDTQuarkusManager.java

```diff
@@ -211,8 +211,15 @@ private static void processConfigProperty(IJavaElement javaElement, IAnnotation
 				String fieldTypeName = getResolvedTypeName(field);
 				IType fieldClass = findType(field.getJavaProject(), fieldTypeName);
 				String defaultValue = getAnnotationMemberValue(configPropertyAnnotation, "defaultValue");
-				addField(field, fieldTypeName, fieldClass, propertyName, defaultValue, converter, javadocCache, null,
-						quarkusProperties, monitor);
+				// Location (JAR, src)
+				IPackageFragmentRoot packageRoot = (IPackageFragmentRoot) javaElement

```

Comment:
can packageroot be null?

Reply:
Never seen the case and all code that I see doesn't check the NPE